Jenkinsfile

```diff
@@ -98,7 +98,7 @@ pipeline {
 
         stage('Distribution Build') {
           steps {
-            sh './gradlew -x test -x sign build publishToMavenLocal distribute'
+            sh './gradlew -x test -x signMavenJavaPublication build publishToMavenLocal distribute'
```

Comment:
Why is this renamed?

Reply:
Gradle does not know sign as a task and suggest signMavenJavaPublication